### PR TITLE
fix: Format files

### PR DESCRIPTION
**Pull request by krishguptadev**
There is a slight issue, the PRs that were made before https://github.com/EddieHubCommunity/LinkFree/pull/743 are not gonna be formatted... Thus I am adding a few changes to this PR to prevent this, as well as format the files. 🙂 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/748"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Optional fields: `links` and `milestones`
 ```json
 {
   "name": "Eddie Jaoude",
+  "type": "personal",
```
Comment (Panquesito7) [inline README.md:56]: For another PR/issue: maybe we should mention about adding communities and personal accounts properly. 🙂

**Review by Panquesito7 — APPROVED** [review]
Overall, all looking good! 🚀

File: README.md (hunk 1 above)
Comment (krishguptadev) [reply README.md:56]: Ah 🤦 this was to be in #749 I'll move the change there...

Comment (Panquesito7) [reply README.md]: > Ah 🤦 this was to be in #749 I'll move the change there...


Alright! But this comment was to make another PR/issue to clarify how this works.

**Review by Panquesito7 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (krishguptadev) [reply README.md]: That will be done... I'll have a convo with Eddie, about where to add it

Comment (krishguptadev) [reply README.md]: and I'll make sure it is added, infact I'll add myself

**Review by Vyvy-vi — APPROVED** [review]
lgtm

**Comment by krishguptadev** [comment]
Hmm, this wasn't meant to merged yet 🙄 I was working on the new suggestion that Eddie gave...

**Comment by krishguptadev** [comment]
I'll make a PR to fix what hasn't been worked on, else it will fail even on good PRs.

**Pull request merged**